bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,87 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
```

Comment:
Doing `import coalib.bearlib.languages.documentation as coalib_documentation` and using that from henceforth maybe a good idea to avoid verbosity.
Personal preference though, so not very important